protocol/x/clob/keeper/msg_server_cancel_orders.go

```diff
@@ -54,7 +54,7 @@ func (k msgServer) CancelOrder(
 					return
```

Comment:
Can you reassign the returned error in this case? Some software deployments might have associated block explorers for viewing TXs, and in this case the user's cancellation technically wasn't received in time rather than order not found (order was already fully-filled / undercollateralized).

So something like this:

err = types.ErrStatefulOrderCancellationFailedForAlreadyRemovedOrder